controllers/replication.storage/volumereplication_controller.go

```diff
@@ -614,6 +644,30 @@ func (r *VolumeReplicationReconciler) enableReplication(vr *volumeReplicationIns
 	return nil
 }
 
+// getVolumeReplicationInfo gets volume replication info.
+func (r *VolumeReplicationReconciler) getVolumeReplicationInfo(vr *volumeReplicationInstance) (*timestamppb.Timestamp, error) {
```

Comment:
this does not return an `info` kind of object? return something like what `volumeReplication.GetInfo()` does, or rename the function to return a LastSyncTimestamp?

Reply:
yeah for now it just returns last sync time, but this name is given just with the thought of future implementation, we may like to return other informations along with the last sync time, hence don't want it to be specific.

Reply:
The current state is confusing, so you will have to decide what this function does. It can be renamed once it gets extended, no need to prepare function names for things in the future. Leave a comment with the possibility if you think it is useful.